tests/test_files/csv.yaml

```diff
@@ -5,6 +5,16 @@ kind: ClusterServiceVersion
 metadata:
   name: marketplace-operator.v0.0.1
   namespace: placeholder
+  annotations:
+    categories: openshift required
+    certified: "true"
+    containerImage: quay.io/openshift/origin-operator-marketplace:latest
+    createdAt: 2019/11/15
```

Comment:
The `createdAt` field should use the `yyyy-mm-ddThh:mm:ssZ` format as described in the marketplace operator repo: https://github.com/operator-framework/operator-marketplace/blob/master/docs/marketplace-required-csv-annotations.md

Reply:
Addressed in latest commit.